templates/footer.php

```diff
@@ -1,5 +1,5 @@
 <footer class="content-info" role="contentinfo">
   <div class="container">
-    <?php dynamic_sidebar('sidebar-footer'); ?>
+    <?php wp_nav_menu( array('container_class' => 'menu-footer', 'menu_class' => 'nav navbar-nav', 'theme_location' => 'secondary_navigation') ); ?>
```

Comment:
This isn't necessary and I think we should remove it. It's a deviation from upstream when upstream has "widgets" and among them is a way to set the footer to a custom menu. I had this working fine.